.github/workflows/development.yaml

```diff
@@ -0,0 +1,124 @@
+name: Development
+on:
+  pull_request:
+  push:
+    tags:
+      - '*.*.*'
+jobs:
+  build:
```

Comment:
`test` for changelog is missing here. We should fail immediately if parsing of the changelog fails so we don't have to wait for builds and future integration tests to run just to fail on changelog parsing.